src/aig/gia/giaMfs.c

```diff
@@ -381,6 +399,15 @@ Gia_Man_t * Gia_ManInsertMfs( Gia_Man_t * p, Sfm_Ntk_t * pNtk, int fAllBoxes )
             continue;
         }
         Vec_IntClear( vLeaves );
+
+        // handle internal CIs first, before we go looking for vLeaves
+        if ( iGroup != -1 && Abc_LitIsCompl(iGroup) )
+        {
+            //Dau_DsdPrintFromTruth( pTruth, Vec_IntSize(vLeaves) );
```

Comment:
maybe this line should be cleaned up.

Reply:
In the ABC coding syle I am not sure this is a line waiting to be cleaned up, rather a line waiting to be uncommented for debugging. Lines like these are everywhere so I don't bother, to be frank

Reply:
Yeah, that's fine.